src/Draco.Compiler/Internal/Binding/Binder_UntypedExpression.cs

```diff
@@ -469,27 +474,66 @@ private UntypedExpression BindMemberExpression(MemberExpressionSyntax syntax, Co
             // Error, don't cascade
             return new UntypedReferenceErrorExpression(syntax, err.Symbol);
         }
-        else if (left is UntypedModuleExpression moduleExpr)
+        Symbol? typ = left is UntypedModuleExpression moduleExpr
```

Comment:
Let's not have cryptic names like this just because type is already taken. In this case let's give one or both some more specific name